test/core/schema.test.js

```diff
@@ -304,7 +304,7 @@ describe("SchemaObject", () => {
           { ...defaults }
         )
       ).to.equal(
-        `(Partial<components["schemas"]["StringType"]>) & (Partial<components["schemas"]["NumberType"]>) & (Partial<components["schemas"]["BooleanType"]>)`
```

Comment:
👍 this seems more correct to me

Reply:
Bah actually, I re-read the spec and I’m second-guessing it now. See other comment.